CorsixTH/Lua/dialogs/menu.lua

```diff
@@ -753,7 +753,7 @@ function UIMenuBar:makeGameMenu(app)
   end
   local function overlay(...)
     local args = {n = select('#', ...), ...}
-    return function(item, m)
+    return function(item, m) -- luacheck: ignore 212
```

Comment:
It's not needed, but was this one meant to have the full inline context as the other 212 ignores have?

Reply:
That function could be ..., I made this change as I understand it. It's local, no context.